config/crd/bases/kmm.sigs.k8s.io_modules.yaml

```diff
@@ -2034,48 +2034,6 @@ spec:
                               description: Regexp is a regular expression to be match
                                 against node kernels.
                               type: string
-                            sign:
```

Comment:
Did you run `make generate` before creating the PR?

Reply:
Hey @ybettan, apologies for my late reply. Yes, I run `make generate && make manifests` and this is what resulted the in the removal of the `sign` sections, as they were not yet part of the CRD.